docs/source/specification/quarterly_roadmap.md

```diff
@@ -21,52 +21,65 @@
 
 A quarterly roadmap will be published to give the DataFusion community visibility into the priorities of the projects contributors. This roadmap is not binding.
 
-## 2022 Q1
+## 2022 Q2
 
 ### DataFusion Core
 
-- Publish official Arrow2 branch
-- Implementation of memory manager (i.e. to enable spilling to disk as needed)
+- IO Improvements
```

Comment:
FYI @tustvold

Reply:
Not entirely sure what this specifically is referring to, but I definitely intend to focus on improving the IO and scheduling stories in arrow-rs and DataFusion. See https://github.com/apache/arrow-rs/issues/1473 and https://github.com/apache/arrow-datafusion/issues/2079. Not sure if we want to explicitly call out the scheduling side of this.

I may also get to proper filter pushdown to parquet if I have time - https://github.com/apache/arrow-rs/issues/1191

Reply:
Thanks @tustvold.  I plan on finishing the work summarized on https://github.com/apache/arrow-datafusion/issues/1777 which is what that refers to